cmd/landscaper-gardenlet/app/landscaper-gardenlet.go

```diff
@@ -0,0 +1,130 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	gardenletconfig "github.com/gardener/gardener/pkg/gardenlet/apis/config"
+	gardenletconfigv1alpha1 "github.com/gardener/gardener/pkg/gardenlet/apis/config/v1alpha1"
+	"github.com/gardener/gardener/pkg/landscaper/gardenlet"
+	"github.com/gardener/gardener/pkg/landscaper/gardenlet/apis/imports"
+	importsv1alpha1 "github.com/gardener/gardener/pkg/landscaper/gardenlet/apis/imports/v1alpha1"
+	importvalidation "github.com/gardener/gardener/pkg/landscaper/gardenlet/apis/imports/validation"
+	"github.com/gardener/gardener/pkg/version/verflag"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+// NewCommandStartLandscaperGardenelet creates a *cobra.Command object with default parameters
+func NewCommandStartLandscaperGardenelet(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "landscaper-gardenlet",
+		Short: "Launch the Gardener landscaper component for the Gardenlet.",
+		Long:  "Gardener landscaper component for the Gardenlet. Sets up the Garden cluster and deploys the Gardenlet with TLS bootstrapping to automatically register the configured Seed cluster.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			verflag.PrintAndExitIfRequested()
+
+			if len(args) != 0 {
+				return errors.New("arguments are not supported. Please only set the path to the configuration file via environment variable \"IMPORTS_PATH\"")
+			}
+
+			return run(ctx)
+		},
+	}
+
+	// add version flag
+	flags := cmd.Flags()
+	verflag.AddFlags(flags)
+	return cmd
+}
+
+func run(ctx context.Context) error {
+	landscaperOperation, importPath, componentDescriptorPath, err := getLandscaperEnvironmentVariables()
+	if err != nil {
+		return err
+	}
+
+	imports, err := loadImportsFromFile(importPath)
+	if err != nil {
+		return fmt.Errorf("unable to load landscaper imports: %v", err)
+	}
+
+	if errs := importvalidation.ValidateLandscaperImport(imports); len(errs) > 0 {
+		return fmt.Errorf("errors validating the landscaper imports: %+v", errs)
+	}
+
+	landscaper, err := gardenlet.NewGardenletLandscaper(imports, landscaperOperation, componentDescriptorPath)
+	if err != nil {
+		return err
+	}
+
+	return landscaper.Run(ctx)
+}
+
+func getLandscaperEnvironmentVariables() (string, string, string, error) {
+	var operation string
+	if operation = os.Getenv("OPERATION"); operation == "" {
+		return "", "", "", fmt.Errorf("environment variable \"OPERATION\" has to be set and must either be \"RECONCILE\" or \"DELETE\"")
+	}
+
+	var importPath string
+	if importPath = os.Getenv("IMPORTS_PATH"); importPath == "" {
```

Comment:
Can you use  https://github.com/gardener/landscaper/blob/efcf0e1f74439d4497161f992bf12a5f4124dc57/pkg/apis/deployer/container/constants.go#L53?